tools/osx/crosstool/wrapped_clang.cc

```diff
@@ -400,7 +402,10 @@ int main(int argc, char *argv[]) {
     }
```

Comment:
Any reason not to get rid of all aborts: lines 171, 353, 399?  Can use exit(EXIT_FAILURE) to short circuit any failure.

Reply:
Done